easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.2.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,198 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.2.0'
```

Comment:
Note to self: update to AlphaFold v2.2.2 which was released very recently...

Reply:
ping @boegel